kippo/accounts/admin.py

```diff
@@ -162,15 +163,24 @@ def get_github_organizations(self, obj):
 class PersonalHolidayAdmin(AllowIsStaffAdminMixin, admin.ModelAdmin):
     list_display = ("user", "is_half", "day", "duration")
 
+    def get_form(self, request, obj=None, **kwargs):
```

Comment:
Can you write a testcase for this?

Confirm that returned form is as expected:

- case where request is a superuser
- case where request is NOT a superuser